### PR TITLE
Completed the Login Page for Pizza Ordering System

### PR DESCRIPTION
**Pull request by VishnuMarontavida**

### CODE REVIEW

File: PizzaOrderingSystem/src/app/effects/pizza-auth.effects.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  //This will trigger for login operation.
+  login$ = createEffect(() => {
+    return this.actions$.pipe(
+      ofType(loginStart),
+      exhaustMap((action: any) => {
+       
+        return this.authService.login(action.userData).pipe(
+          map((data) => {
+            this.store.dispatch(setLoadingSpinner({ status: false }));
+            this.store.dispatch(setErrorMessage({ message: '' }));
```
Comment (MarcoHuib) [inline PizzaOrderingSystem/src/app/effects/pizza-auth.effects.ts]: It's not a best practice to dispatch multiple actions from an effect. What you should do is return the success or failed action from the effect. And catch this action inside the spinner reducer.

https://twitter.com/mikeryandev/status/938108618133602304

File: PizzaOrderingSystem/angular.json [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        }
+      },
+      "root": "",
+      "sourceRoot": "src",
+      "prefix": "app",
+      "architect": {
+        "build": {
+          "builder": "@angular-devkit/build-angular:browser",
+          "options": {
+            "outputPath": "dist/PizzaOrderingSystem",
```
Comment (DWijnands) [inline PizzaOrderingSystem/angular.json:23]: test

**Review by DWijnands — COMMENTED** [review]
test

File: PizzaOrderingSystem/src/app/effects/pizza-auth.effects.ts (hunk 1 above)
Comment (VishnuMarontavida) [reply PizzaOrderingSystem/src/app/effects/pizza-auth.effects.ts]: Removed the multiple actions from an effect, and Pushed the code to the server feature/login-page

**Comment by VishnuMarontavida** [comment]
Taken care the pull request comment and merged with the base branch

**Pull request merged**